.github/workflows/testing.yml

```diff
@@ -55,9 +68,20 @@ jobs:
           && break \
         ; done)
 
-        
-    - name: coveralls
-      if: ${{ always() && github.event_name == 'push' && github.ref == 'refs/heads/develop' }}
+    - name: Run tests in parallel
+      if: ${{ always() && inputs.xdist == 'yes' }}
+      timeout-minutes: 25
```

Comment:
ah, if condition :)

we should try again if -n auto takes more than 2. So that we can take github management.
Last time I fixated it to 6, because auto seems not to work on that platform

Reply:
the "steps:" is a keyword, we can't change, but the id: "lable" should be something we could give a meaningful name. In https://docs.github.com/en/actions/learn-github-actions/contexts a lable of "github_context_step" is used. 
A rename is not so important for me now,